src/main/java/com/api/Comment.java

```diff
@@ -0,0 +1,59 @@
+package com.api;
+
+import com.google.api.client.util.DateTime;
+
+public class Comment {
```

Comment:
Why not use Lombok annotation here. Also, you could create a builder and use it.

Reply:
May I use Lombok annotation next time, please?